ONSdigital/eq-author-app
# Ear 298 route on or checkbox option

### What is the context of this PR?

Previously, when you try to build a routing rule on a checkbox answer which matches on any of the selected options, and one of those options is a mutually exclusive or option, it didn't work when you tried it in Runner.

### How to review 

Try and build a routing rule which routes based on the 'Any of' condition on a checkbox answer with one of the options being a mutually exclusive or option. Try running it in Runner; if it works, grand.

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
